scss/atoms/input_styles/_input_styles.scss

```diff
@@ -0,0 +1,116 @@
+/*
+* ==========================================================================
+* Input field styles
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis";
+
+/*
+* ==========================================================================
+* Normal state styling
+* ==========================================================================
+*/
+
+
+input[type=text],
+input[type=url],
+input[type=tel],
+input[type=number],
+input[type=color],
+input[type=email],
+input[type=search],
+input[type=password] {
+	border-radius: 2px;
+	border: 1px solid map-get($text, light);
+	box-shadow: inset 0 1px 3px  rgba(0,0,0,.24);
```

Comment:
Working exactly on that, wasn't aware of the way variables are used. Looking at the button example gave the idea.